src/libraries/System.Reflection.Emit/tests/TypeBuilder/TypeBuilderGetConstructor.cs

```diff
@@ -8,14 +8,14 @@ namespace System.Reflection.Emit.Tests
     public class TypeBuilderGetConstructor
     {
         [Fact]
-        public void GetConstructor_DeclaringTypeOfConstructorNotGenericTypeDefinition_ThrowsArgumentException()
+        public void GetConstructor_DeclaringTypeOfConstructorNotGenericTypeDefinition()
```

Comment:
Can you make sure the cases where `ArgumentException` are thrown from the method are covered in tests? That way we know they are working correctly.

Reply:
I see a bunch of tests below have

```
[ActiveIssue("[https://github.com/dotnet/runtime/issues/2389](https://github.com/dotnet/runtime/issues/2389)", TestRuntimes.Mono)]
```

Are any of these fixed by your changes?

Reply:
I've just pushed all corrections (very sorry for the delays, I have very busy days recent times...)
Let's see which of the marked tests will pass and compare results against those reported in the mentioned issue.

Reply:
No need to apologize. We totally understand if people are busy.

Looks like the tests are now failing:

```
 Starting:    System.Reflection.Emit.Tests (parallel test collections = on, max threads = 2)
    System.Reflection.Emit.Tests.TypeBuilderGetConstructor.GetConstructor_DeclaringTypeOfConstructorWithoutGenericParameters_ThrowsArgumentException [FAIL]
      Assert.Equal() Failure
                 (pos 0)
      Expected: type
      Actual:   constructor
                 (pos 0)
      Stack Trace:
        /_/src/libraries/Common/tests/TestUtilities/System/AssertExtensions.cs(123,0): at System.AssertExtensions.Throws[T](String expectedParamName, Func`1 testCode)
        /_/src/libraries/System.Reflection.Emit/tests/TypeBuilder/TypeBuilderGetConstructor.cs(16,0): at 
```